bedevere/stage.py

```diff
@@ -227,6 +236,27 @@ async def new_review(event, gh, *args, **kwargs):
             raise ValueError(f"unexpected review state: {state!r}")
 
 
+@router.register("pull_request_review", action="dismissed")
+async def dismissed_review(event, gh, *args, **kwargs):
+    """Update the stage based on a dismissed review."""
+    pull_request = event.data["pull_request"]
+
+    # Poor-man's asynchronous any().
+    async for _ in core_dev_reviewers(gh, pull_request["url"]):
+        # No need to update the label as there is still a core dev review.
+        return
+    else:
```

Comment:
This is else is necessary for coverage on `3.9`. On `3.10` and `3.8` this works without, but on `3.9` `coverage` complains about a missing branch.